src/barkhaPJun20/SumOfDigitsInString.java

```diff
@@ -0,0 +1,31 @@
+package barkhaPJun20;
+
+public class SumOfDigitsInString {
+	
+	
+	void addition(String name) {
```

Comment:
@BarkhaPJun20 
Code logic up the mark, keep it up.
Give method name proper way. (work on this)

Reply:
Updated proper method name.